src/uct/api/v2/uct_v2.h

```diff
@@ -378,6 +380,42 @@ typedef struct uct_md_mem_reg_params {
      * Operation specific flags, using bits from @ref uct_md_mem_flags_t.
      */
     uint64_t                     flags;
+
+    /**
+     * dmabuf file descriptor of the memory region to register.
+     *
+     * If is set (along with its corresponding bit in the field_mask -
+     * @ref UCT_MD_MEM_REG_FIELD_DMABUF_FD), the memory region will be
+     * registered using dmabuf mechanism. In this case, the address argument
+     * provided to the registration function is the remote access virtual
+     * address of the memory region (for RMA or AMO operation from a peer
+     * endpoint), and the local address to register is deduced from
```

Comment:
@yosefe  maybe add ref to the function itself. 

I'm still puzzled why local function registration requires target/remote VA ? I was try to trace it down in the code and it seems like it is not used for dmabuf flow.

Reply:
As I read it here https://github.com/linux-rdma/rdma-core/blob/6c8b24fb64770449d76193f8dead3e13a356ebe6/libibverbs/man/ibv_reg_mr.3#L89

I don't see reference for remote addr

Reply:
Oh ... "remote access virtual" - I think you don't mean "remote" here. It is really local VA that is shared with remote peers within the range of this DMA buffer.

Reply:
"In this case, the address argument provided to the registration function is the remote access virtual address of the memory region (for RMA or AMO operation from a peer endpoint), and the local address to register is deduced from"
->
"In this case, the address argument provided to the registration function is the local virtual address of the memory region that will be used for an access (RMA or AMO) by peer endpoints"

Reply:
So, you just nuked the entire paragraph ?
```
In this case, the address argument
     * provided to the registration function is the remote access virtual
     * address of the memory region (for RMA or AMO operation from a peer
     * endpoint), and the local address to register is deduced from
     * @ref uct_md_mem_reg_params_t.dmabuf_offset field.
```

Reply:
@shamisp yes, it was replaced by comment in dmabuf_offset